lab_3_ann_retriever/main.py

```diff
@@ -304,6 +415,15 @@ def retrieve_vectorized(self, query_vector: Vector) -> str | None:
 
         In case of corrupt input arguments, None is returned.
         """
+        if not isinstance(query_vector, tuple):
+            return None
+        for vector in self._document_vectors:
+            if len(vector) < len(query_vector):
```

Comment:
why are you checking like this? what will be if `len(vector > len(query_vector)` ?? nothing?

Reply:
yeah, I understood and fixed this